packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+# Organizing Communications
+
+# Introduction
+
+Communications in a healthcare context (modeled as [Communication](/docs/api/fhir/resources/communication) resources) can include many scenarios – patient to physician, physician to physician, device to physician, and more. This results in a large volume of communications within this context, so ensuring that they are well organized is vital. For example, a provider may have staff on call to respond to patient messages. Without proper organization, messages may not reach the correct staff member.
+
+This guide covers different ways that you can organize communications and how to think about different properties on the `Communication` element. It will cover:
+
+- Common communication organization patterns, including `Communication.category`, `Communication.topic`, and threading.
+- Searching for communications and threads using the the search helper method from the [Medplum Client SDK](/docs/sdk/classes/MedplumClient)
```

Comment:
Combine this with the "sorting" and "querying" sections. It should just show 1 example, that shows how to use REST or GraphQL to: 1) retrieve all threads 2) Retrieve all messages in a particular thread, and sort them chronological order